readme.md

```diff
@@ -158,7 +158,7 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Question:** At least one of these will throw an error. Which one? How can you get around this issue?
 
-* **Your Answer:** 
+* **Your Answer:** For put you can just search for the updated item
```

Comment:
You should encounter an error when you run `.create().select()`. You will have to re-request the information or use the `.save()` method to return the updated record.